The retry GitHub action is failing

**What were you trying to achieve?**
- Run a yaml file update using the [Wandalen/wretry.action](https://github.com/Wandalen/wretry.action) GitHub action

**What are the expected results?**
- Retries to be done without fail

**What are the received results?**
- The action kept failing with the error below:
```

Run Wandalen/wretry.action@v1.1.0
Run Wandalen/wretry.action@v1.1.0_js_action
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'uuid'
Require stack:
- /home/runner/work/_actions/Wandalen/wretry.action/v1.1.0_js_action/node_modules/@actions/core/lib/file-command.js
- /home/runner/work/_actions/Wandalen/wretry.action/v1.1.0_js_action/node_modules/@actions/core/lib/core.js
- /home/runner/work/_actions/Wandalen/wretry.action/v1.1.0_js_action/src/Pre.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/runner/work/_actions/Wandalen/wretry.action/v1.1.0_js_action/node_modules/@actions/core/lib/file-command.js:28:16)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) ***
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/Wandalen/wretry.action/v1.1.0_js_action/node_modules/@actions/core/lib/file-command.js',
    '/home/runner/work/_actions/Wandalen/wretry.action/v1.1.0_js_action/node_modules/@actions/core/lib/core.js',
    '/home/runner/work/_actions/Wandalen/wretry.action/v1.1.0_js_action/src/Pre.js'
  ]
***
```

**What are the steps to reproduce the issue?**
-  Run any workflow that invloves use of the retry action

**Additional context**
- The GitHub action homepage is [here](https://github.com/Wandalen/wretry.action)
- There has been a similar reported issue [here](https://github.com/Wandalen/wretry.action/issues/93)
- An example of where we use the action is:
  - https://github.com/airqo-platform/AirQo-frontend/blob/bcc1d0674a500f84c77445d42e6609c5b411d44e/.github/workflows/deploy-frontends-to-staging.yml#L137

--- a/.github/workflows/deploy-frontends-to-production.yml
+++ b/.github/workflows/deploy-frontends-to-production.yml
@@ -94,7 +94,7 @@ jobs:
           docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-platform-frontend:latest
 
       - name: Update corresponding helm values file(with retry)
-        uses: Wandalen/wretry.action@master # Retries action on fail
+        uses: Wandalen/wretry.action@v1.0.36 # Retries action on fail
         with:
           action: fjogeleit/yaml-update-action@main # Action to retry
           with: |
@@ -245,7 +245,7 @@ jobs:
           docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-calibrate-app:latest
 
       - name: Update corresponding helm values file(with retry)
-        uses: Wandalen/wretry.action@master # Retries action on fail
+        uses: Wandalen/wretry.action@v1.0.36 # Retries action on fail
         with:
           action: fjogeleit/yaml-update-action@main # Action to retry
           with: |
@@ -304,7 +304,7 @@ jobs:
           docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-next-platform:latest
 
       - name: Update corresponding helm values file(with retry)
-        uses: Wandalen/wretry.action@master # Retries action on fail
+        uses: Wandalen/wretry.action@v1.0.36 # Retries action on fail
         with:
           action: fjogeleit/yaml-update-action@main # Action to retry
           with: |
@@ -352,7 +352,7 @@ jobs:
           docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-prod-docs:latest
 
       - name: Update corresponding helm values file(with retry)
-        uses: Wandalen/wretry.action@master # Retries action on fail
+        uses: Wandalen/wretry.action@v1.0.36 # Retries action on fail
         with:
           action: fjogeleit/yaml-update-action@main # Action to retry
           with: |

--- a/.github/workflows/deploy-frontends-to-staging.yml
+++ b/.github/workflows/deploy-frontends-to-staging.yml
@@ -134,7 +134,7 @@ jobs:
           docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-platform-frontend:latest
 
       - name: Update corresponding helm values file(with retry)
-        uses: Wandalen/wretry.action@master # Retries action on fail
+        uses: Wandalen/wretry.action@v1.0.36 # Retries action on fail
         with:
           action: fjogeleit/yaml-update-action@main # Action to retry
           with: |
@@ -282,7 +282,7 @@ jobs:
           docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-calibrate-app:latest
 
       - name: Update corresponding helm values file(with retry)
-        uses: Wandalen/wretry.action@master # Retries action on fail
+        uses: Wandalen/wretry.action@v1.0.36 # Retries action on fail
         with:
           action: fjogeleit/yaml-update-action@main # Action to retry
           with: |
@@ -341,7 +341,7 @@ jobs:
           docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-next-platform:latest
 
       - name: Update corresponding helm values file(with retry)
-        uses: Wandalen/wretry.action@master # Retries action on fail
+        uses: Wandalen/wretry.action@v1.0.36 # Retries action on fail
         with:
           action: fjogeleit/yaml-update-action@main # Action to retry
           with: |
@@ -389,7 +389,7 @@ jobs:
           docker push ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-docs:${{ needs.image-tag.outputs.build_id }}
 
       - name: Update corresponding helm values file(with retry)
-        uses: Wandalen/wretry.action@master # Retries action on fail
+        uses: Wandalen/wretry.action@v1.0.36 # Retries action on fail
         with:
           action: fjogeleit/yaml-update-action@main # Action to retry
           with: |

